sway/sway.5.scd

```diff
@@ -51,7 +51,7 @@ The following commands may only be used in the configuration file.
 	*wordexp*(3) for details). The same include file can only be included once;
 	subsequent attempts will be ignored.
 
-*set* <name> <value>
+*set* $<name> <value>
```

Comment:
The preferred format is without the symbol

Reply:
It seems the symbol is used in the default config, so I guess we'll just go with that.